Extensions/Xtensive.Orm.Localization.Tests/ProjectionToCustomTypesTests.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Xtensive.Orm.Localization.Tests.Model;
+
+namespace Xtensive.Orm.Localization.Tests
+{
+  [TestFixture]
+  public class ProjectionToCustomTypesTests : AutoBuildTest
+  {
+    protected override void PopulateDatabase()
+    {
+      using (var session = Domain.OpenSession()) {
+        using (var ts = session.OpenTransaction()) {
+
+          // populating database
+          var m1 = new Country(session) {
+            Identifier = "HUN",
+            Name = "Magyarország"
+          };
+          var m2 = new Country(session) {
+            Identifier = "RUS",
+            Name = "Oroszország"
+          };
+          using (new LocalizationScope(EnglishCulture))
+            m2.Name = "Russia";
+          using (new LocalizationScope(SpanishCulture))
+            m2.Name = "Rusia";
```

Comment:
Use brackets for ```using``` body. There is a rule for it in ```.editorconfig``` file so VS should advice you to add brackets. Make sure that the file is loaded by IDE

Reply:
done